### PR TITLE
[tensorflow] AFL++ build

### PR DESCRIPTION
**Pull request by kobrineli**

### CODE REVIEW

File: projects/tensorflow/make_corporas.sh [hunk 1]
```diff
@@ -0,0 +1,7 @@
+#!/bin/bash
```
Comment (SweetVishnya) [inline projects/tensorflow/make_corporas.sh:1]: Remove this file

**Review by SweetVishnya — CHANGES_REQUESTED** [review]

File: projects/tensorflow/make_corporas.sh (hunk 1 above)
Comment (kobrineli) [reply projects/tensorflow/make_corporas.sh]: done

File: projects/tensorflow/build_afl.sh [hunk 2]
```diff
@@ -0,0 +1,174 @@
+#!/bin/bash -eu
+# Copyright 2018 Google Inc.
+# Modifications copyright (C) 2021 ISP RAS
```
[the remark this answers is not in the document]
Comment (kobrineli) [reply projects/tensorflow/build_afl.sh]: done

File: projects/tensorflow/Dockerfile [hunk 3]
```diff
@@ -85,3 +95,16 @@ COPY build_cov.sh /
...
+COPY arg_def_case.zip base64.zip check_numerics.zip cleanpath.zip consume_leading_digits.zip decode_compressed.zip \
+     encode_base64.zip encode_jpeg.zip example_proto_fast_parsing.zip joinpath.zip one_hot.zip parse_tensor_op.zip \
+     parseURI.zip status_group.zip status.zip stringprintf.zip string_replace.zip tstring.zip /
+
+WORKDIR /corpus
+RUN unzip /arg_def_case.zip && unzip /base64.zip && unzip /check_numerics.zip && unzip /cleanpath.zip && \
+    unzip /consume_leading_digits.zip && unzip /decode_compressed.zip && unzip /encode_base64.zip && \
+    unzip /encode_jpeg.zip && unzip /example_proto_fast_parsing.zip && unzip /joinpath.zip && \
+    unzip /one_hot.zip && unzip /parse_tensor_op.zip && unzip /parseURI.zip && unzip /status_group.zip && \
+    unzip /status.zip && unzip /stringprintf.zip && unzip /string_replace.zip && unzip /tstring.zip
```
[the remark this answers is not in the document]
Comment (kobrineli) [reply projects/tensorflow/Dockerfile:108]: done

[the remark this answers is not in the document]
Comment (SweetVishnya) [reply projects/tensorflow/Dockerfile:108]: rm them in the same command to reduce space used by docker

[the remark this answers is not in the document]
Comment (kobrineli) [reply projects/tensorflow/Dockerfile]: done

**Review by SweetVishnya — APPROVED** [review]
We'll merge it as soon as your fix PR is in AFL++ stable.

**Pull request merged**